examples/blog/README.md

```diff
@@ -41,6 +42,8 @@ Astro looks for `.astro` or `.md` files in the `src/pages/` directory. Each page
 
 There's nothing special about `src/components/`, but that's where we like to put any Astro/React/Vue/Svelte/Preact components.
 
+The `src/content/` directory contains "collections" of related Markdown and MDX documents. Astro will generate a `getCollection()` function to retrieve posts from `src/content/blog/`, and type-check your frontmatter using an optional schema. See [Astro's Content Collections docs](https://docs.astro.build/en/guides/content-collections/) to learn more.
```

Comment:
```suggestion
The `src/content/` directory contains "collections" of related Markdown and MDX documents. Use `getCollection()` to retrieve posts from `src/content/blog/`, and type-check your frontmatter using an optional schema. See [Astro's Content Collections docs](https://docs.astro.build/en/guides/content-collections/) to learn more.
```
For your consideration: Maybe more user-focused? Less about "What Astro is doing under the hood."

I also like the explicitness of telling them to USE that function since some might already be used to using `Astro.glob()` and this hits them with New Usage Instructions.